src/reports/models/reportsManager.js

```diff
@@ -65,9 +63,10 @@ module.exports.deleteReport = async (testId, reportId) => {
     await databaseConnector.deleteReport(testId, reportId);
 };
 
-module.exports.postReport = async (testId, reportBody) => {
-    const startTime = new Date(Number(reportBody.start_time));
-    const job = await jobConnector.getJob(reportBody.job_id);
+module.exports.postReport = async(runId, testId, jobId, startTime) => {
+    const reportId = runId;
+    const test = await testsManager.getTest(testId);
```

Comment:
this isn't needed and is another extra call to the DB. In the `postReport` flow you'll see that we already retreive the `test` object and pass only its `id` to the subsequent functions. Lets instead pass the `test` object in the function instead of `test.id`